Action/src/Action.java

```diff
@@ -0,0 +1,109 @@
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+public class Action {
+    public static void main(String[] args) throws IOException {
+        int countOfWater = 400;
+        int countOfMilk = 540;
+        int countOfCoffeeBeans = 120;
+        int countOfMoney = 550;
+        int countOfCups = 9;
+
+        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
+        System.out.println("The coffee machine has:");
+        System.out.println(countOfWater + " of water");
+        System.out.println(countOfMilk + " of milk");
+        System.out.println(countOfCoffeeBeans + " of coffee beans");
+        System.out.println(countOfMoney + " of money");
+        System.out.println("Write action (buy, fill, take):");
+        System.out.print(">");
+        String action = reader.readLine();
+
+        if(action.equals("buy"))
+            buy(countOfWater,countOfMilk,countOfCoffeeBeans,countOfMoney,countOfCups);
+        if(action.equals("fill"))
+            fill(countOfWater,countOfMilk,countOfCoffeeBeans,countOfMoney,countOfCups);
+        if(action.equals("take"))
+            take(countOfWater,countOfMilk,countOfCoffeeBeans,countOfMoney,countOfCups);
+
+        reader.close();
+
+    }
+    public static void buy(int countOfWater, int countOfMilk,int countOfCoffeeBeans, int countOfMoney,int countOfCups) throws IOException{
+        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
+        System.out.println("What do you want to buy? 1 - espresso, 2 - latte, 3 - cappuccino:");
+        System.out.print(">");
+        String number = reader.readLine();
+        System.out.println(number);
+        if(number.equals("1")){
```

Comment:
you can convert String to int, and maybe use a 'switch' case

Reply:
done